### ISSUE TITLE
Bug: Doesn't run with deps.edn

### ISSUE THREAD

**Issue by kxygk** [boechat107/eye-boof#6]
Seems like there is a bug in the `(ns ...)` declaration.

``` clojure

(ns eye-boof.visualization
  "Provides functions to visualize images using JFrame."
  (require [eye-boof.core.io :refer [resource->buff-image]])
  (:import boofcv.gui.image.ShowImages
           boofcv.gui.ListDisplayPanel
           boofcv.gui.binary.VisualizeBinaryData
           boofcv.struct.image.ImageBase))

```

The `require` should read `:require`

The error I get is:

```
  Show: Project-Only All 
  Hide: Clojure Java REPL Tooling Duplicates  (23 frames hidden)

2. Unhandled clojure.lang.Compiler$CompilerException
   Error compiling eye_boof/visualization.clj at (1:1)
   #:clojure.error{:phase :macro-syntax-check,
                   :line 1,
                   :column 1,
                   :source "eye_boof/visualization.clj",
                   :symbol clojure.core/ns}
             Compiler.java: 6972  clojure.lang.Compiler/checkSpecs
             Compiler.java: 6988  clojure.lang.Compiler/macroexpand1
             Compiler.java: 7075  clojure.lang.Compiler/macroexpand
             Compiler.java: 7161  clojure.lang.Compiler/eval
             Compiler.java: 7636  clojure.lang.Compiler/load
                   RT.java:  381  clojure.lang.RT/loadResourceScript
                   RT.java:  372  clojure.lang.RT/loadResourceScript
                   RT.java:  459  clojure.lang.RT/load
                   RT.java:  424  clojure.lang.RT/load
                  core.clj: 6126  clojure.core/load/fn
                  core.clj: 6125  clojure.core/load
                  core.clj: 6109  clojure.core/load
               RestFn.java:  408  clojure.lang.RestFn/invoke
                  core.clj: 5908  clojure.core/load-one
                  core.clj: 5903  clojure.core/load-one
                  core.clj: 5948  clojure.core/load-lib/fn
                  core.clj: 5947  clojure.core/load-lib
                  core.clj: 5928  clojure.core/load-lib
               RestFn.java:  142  clojure.lang.RestFn/applyTo
                  core.clj:  667  clojure.core/apply
                  core.clj: 5985  clojure.core/load-libs
                  core.clj: 5969  clojure.core/load-libs
               RestFn.java:  137  clojure.lang.RestFn/applyTo
                  core.clj:  667  clojure.core/apply
                  core.clj: 6007  clojure.core/require
                  core.clj: 6007  clojure.core/require
               RestFn.java:  421  clojure.lang.RestFn/invoke
                  core.clj:    1  eyebeef.core/eval9068/loading--auto--
                  core.clj:    1  eyebeef.core/eval9068
                  core.clj:    1  eyebeef.core/eval9068
             Compiler.java: 7177  clojure.lang.Compiler/eval
             Compiler.java: 7166  clojure.lang.Compiler/eval
             Compiler.java: 7636  clojure.lang.Compiler/load
                      REPL:    1  user/eval9064
                      REPL:    1  user/eval9064
             Compiler.java: 7177  clojure.lang.Compiler/eval
             Compiler.java: 7132  clojure.lang.Compiler/eval
                  core.clj: 3214  clojure.core/eval
                  core.clj: 3210  clojure.core/eval
                  main.clj:  437  clojure.main/repl/read-eval-print/fn
                  main.clj:  437  clojure.main/repl/read-eval-print
                  main.clj:  458  clojure.main/repl/fn
                  main.clj:  458  clojure.main/repl
                  main.clj:  368  clojure.main/repl
               RestFn.java: 1523  clojure.lang.RestFn/invoke
    interruptible_eval.clj:   79  nrepl.middleware.interruptible-eval/evaluate
    interruptible_eval.clj:   55  nrepl.middleware.interruptible-eval/evaluate
    interruptible_eval.clj:  142  nrepl.middleware.interruptible-eval/interruptible-eval/fn/fn
                  AFn.java:   22  clojure.lang.AFn/run
               session.clj:  171  nrepl.middleware.session/session-exec/main-loop/fn
               session.clj:  170  nrepl.middleware.session/session-exec/main-loop
                  AFn.java:   22  clojure.lang.AFn/run
               Thread.java:  834  java.lang.Thread/run

1. Caused by clojure.lang.ExceptionInfo
   Call to clojure.core/ns did not conform to spec.
   #:clojure.spec.alpha{:problems
                        [{:path [],
                          :reason "Extra input",
                          :pred
                          (clojure.spec.alpha/cat
                           :attr-map
                           (clojure.spec.alpha/? clojure.core/map?)
                           :ns-clauses
                           :clojure.core.specs.alpha/ns-clauses),
                          :val
                          ((require
                            [eye-boof.core.io
                             :refer
                             [resource->buff-image]])
                           (:import
                            boofcv.gui.image.ShowImages
                            boofcv.gui.ListDisplayPanel
                            boofcv.gui.binary.VisualizeBinaryData
                            boofcv.struct.image.ImageBase)),
                          :via [:clojure.core.specs.alpha/ns-form],
                          :in [2]}],
                        :spec
                        #object[clojure.spec.alpha$regex_spec_impl$reify__2509 0x201390ff "clojure.spec.alpha$regex_spec_impl$reify__2509@201390ff"],
                        :value
                        (eye-boof.visualization
                         "Provides functions to visualize images using JFrame."
                         (require
                          [eye-boof.core.io
                           :refer
                           [resource->buff-image]])
                         (:import
                          boofcv.gui.image.ShowImages
                          boofcv.gui.ListDisplayPanel
                          boofcv.gui.binary.VisualizeBinaryData
                          boofcv.struct.image.ImageBase)),
                        :args
                        (eye-boof.visualization
                         "Provides functions to visualize images using JFrame."
                         (require
                          [eye-boof.core.io
                           :refer
                           [resource->buff-image]])
                         (:import
                          boofcv.gui.image.ShowImages
                          boofcv.gui.ListDisplayPanel
                          boofcv.gui.binary.VisualizeBinaryData
                          boofcv.struct.image.ImageBase))}
```

If you need a minimal example I can upload one :)

**Comment by vgrocha** [comment]
Should be fixed in the `2.2.1-SNAPSHOT` version

**Comment by kxygk** [comment]
oh wow. fantastic. That actually got things working :)
And thank you so much for the quick turn around!
